src/praisonai-agents/praisonaiagents/skills/manager.py

```
@@ -243,6 +243,315 @@ def load_resources(self, name: str) -> bool:
+        )
+        
+        try:
+            with os.fdopen(temp_fd, 'w', encoding='utf-8') as f:
+                f.write(content)
+            # Atomic rename
+            os.replace(temp_path, file_path)
+        except:
+            # Clean up temp file if something failed
+            try:
+                os.unlink(temp_path)
+            except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)